api/app/controllers/TestsController.scala

```diff
@@ -5,14 +5,27 @@ import play.api._
 import play.api.mvc._
 import play.api.libs.json._
 
-import automat.models.{Test}
+import automat.models.{TestsStore}
 
 @Singleton
 class TestsController @Inject() (val controllerComponents: ControllerComponents)
     extends BaseController {
 
-  def index() = Action { implicit request: Request[AnyContent] =>
-    val resp = Json.toJson(Test.exampleTests)
+  def index(slotId: String) = Action { implicit request: Request[AnyContent] =>
+    val resp = Json.toJson(TestsStore.getBySlotId(slotId))
     Ok(resp)
   }
+
+  def create(slotId: String) = Action { implicit request: Request[AnyContent] =>
+    ???
+  }
+
+  def update(testId: String) = Action { implicit request: Request[AnyContent] =>
+    ???
+  }
+
+  def delete(testId: String) = Action { implicit request: Request[AnyContent] =>
+    TestsStore.delete(testId)
+    NoContent
```

Comment:
Probably want to tie the result here to whether or not the delete was successful. I.e. deleting a non-existent test ID should probably return an error response rather than 204.